front/components/assistant/conversation/AgentSuggestion.tsx

```diff
@@ -1,137 +1,141 @@
-import { Button, RobotIcon } from "@dust-tt/sparkle";
+import { AssistantPreview, Button, RobotIcon, Spinner } from "@dust-tt/sparkle";
 import type {
   LightAgentConfigurationType,
   UserMessageType,
   WorkspaceType,
 } from "@dust-tt/types";
-import { useContext, useState } from "react";
+import { useContext, useMemo, useState } from "react";
 
 import { AssistantPicker } from "@app/components/assistant/AssistantPicker";
 import { SendNotificationsContext } from "@app/components/sparkle/Notification";
-import { compareAgentsForSort } from "@app/lib/assistant";
 import { useSubmitFunction } from "@app/lib/client/utils";
 import { useAgentConfigurations } from "@app/lib/swr";
 
 interface AgentSuggestion {
   conversationId: string;
-  latestMentions: string[];
   owner: WorkspaceType;
   userMessage: UserMessageType;
 }
 
 export function AgentSuggestion({
   conversationId,
-  latestMentions,
   owner,
   userMessage,
 }: AgentSuggestion) {
   const { agentConfigurations } = useAgentConfigurations({
     workspaceId: owner.sId,
     agentsGetView: { conversationId: conversationId },
+    includes: ["authors", "usage"],
```

Comment:
Do we need to limit here?

Reply:
Thing if we limit is that some assistants might not be searchable, which would be a quite bad UX